### ISSUE TITLE
Function IoU - Input Expectations

### ISSUE THREAD

**Issue by XDynames** [PyTorchLightning/pytorch-lightning#4244]
## 📚 Documentation

Changing input description to include that predictions should be an integer mask rather than raw logits [b, h, w] vs. [b, c, h, w]

If logits are provided solution runs with warning about number of classes being set incorrectly - from feeding the returned determined num_classes from get_num_classes() to stat_scores_multiple_classes() where it is called again with the now incorrect number of classes.

**Comment by ddrevicky** [comment]
You can assign this to me.
